docs/docs/concepts/infrastructure-architecture.mdx

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 1
+---
+
+# Infrastructure Architecture
+
+JHub Apps, also known as JupyterHub Apps Launcher, is an [Externally-Managed JupyterHub service](https://jupyterhub.readthedocs.io/en/stable/reference/services.html#externally-managed-services) designed to enable users to launch various server types including API services, dashboards such as Panel, Bokeh, Streamlit, standard JupyterLab instances, and any generic Python command.
+
```

Comment:
```suggestion
## Overview of how JHub Apps works within JupyterHub
```
Given my comment below, consider adding a few extra headers to organize the topics you're covering in this first part. (I'm just guessing at this title, could be "High Level Overview" - you should change it as needed).